packages/base/markdown.gts

```diff
@@ -3,13 +3,8 @@ import { BoxelInput } from '@cardstack/boxel-ui/components';
 import { marked } from 'marked';
 import { sanitizeHtml } from '@cardstack/runtime-common';
 
-const markdownOpts = {
-  mangle: false,
-  headerIds: false,
-};
-
 function toHtml(markdown: string | null) {
-  return markdown ? sanitizeHtml(marked(markdown, markdownOpts)) : '';
+  return markdown ? sanitizeHtml(marked(markdown) as string) : '';
```

Comment:
Why do we have to cast this? What type does TypeScript think it is?

Reply:
return type says `string | Promise<string>`

Reply:
I'm curious why it might be a promise. sounds like there is a potential we might be leaking async here unless we await...

Reply:
I think it's only async if the options are set to be { async: true } or if any of the options return async. Here's what the docs say: https://marked.js.org/using_pro#async

Here are the signatures:
<img width="996" alt="types" src="https://github.com/cardstack/boxel/assets/16160806/ada05191-3090-43ca-baee-cb811abf8efb">

Thoughts? Just setting the async options to explicitly be false didn't work for glint.

Reply:
^ @habdelra @lukemelia